sanity-io/sdk
# chore: add test coverage

### Description
This PR adds a `test:coverage` command that runs the Vitest's with the `--coverage` flag using the default (v8) coverage tool.

Note: adding a threashold and running this in the github workflow will come in later PRs.

### What to review
Check that test coverage is shown and the correct files are included/exclude in the coverage reports.

### Testing
1. Run `pnpm run test:coverage`
2. Check the terminal output
3. Check the `/coverage` folder outputs


### Fun gif
![covered](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExOHR2aWk3MTR6eXVpaG9nOXNrc3VpZDR2YWxlMjV3czFzMWFmZWtydiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/x6DPdnZkG5Fls2d6eF/giphy-downsized.gif)
